inc/class-core-sitemaps-index.php

```diff
@@ -53,7 +53,14 @@ public function get_sitemap_list() {
 			array_push( $sitemaps, ...$provider->get_sitemap_entries() );
 		}
 
-		return $sitemaps;
+		/**
+		 * Filters the array of URLs for a sitemap index before rendering.
+		 *
+		 * @since 5.5.0
+		 *
+		 * @param array $sitemaps Array of URLs for a sitemap index.
+		 */
+		return apply_filters( 'core_sitemaps_sitemap_list', $sitemaps );
```

Comment:
In case someone wanted to add back `lastmod` to the index

Reply:
It would be great to document how developers could do this or add `changefreq` or `priority`.

Reply:
Docs added to the readme!